Files/Views/YourHome.xaml.cs

```diff
@@ -188,7 +201,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
         //       This IDisposable.Dispose() needs to be called to unhook events in BundlesWidget to avoid memory leaks.
         public void Dispose()
```

Comment:
Actually this Dispose() will never be called as the comment says.
I'll see what could be done in my next PR :)